packages/libs/e2e-test-utils/src/lib/env.ts

```diff
@@ -25,7 +25,7 @@ const baseEnvSchema = {
   TEST_FUNDER_PRIVATE_KEY: z.string(),
   TEST_APP_MANAGER_PRIVATE_KEY: z.string(),
   TEST_APP_DELEGATEE_PRIVATE_KEY: z.string(),
-  TEST_AGENT_WALLET_PKP_OWNER_PRIVATE_KEY: z.string(),
+  TEST_PLATFORM_USER_WALLET_OWNER_PRIVATE_KEY: z.string(),
```

Comment:
just for clarity, the `TEST_PLATFORM_USER_WALLET_OWNER_PRIVATE_KEY` is the user wallet / PKP, not the agent PKP, right?